service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/models/DigitalTwinMetadata.java

```diff
@@ -0,0 +1,61 @@
+package com.microsoft.azure.sdk.iot.service.digitaltwin.models;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
These and other classes should be in helper directory now that I think more about it.